defs.bzl

```diff
@@ -23,4 +23,5 @@ load("//internal:node.bzl",
 load("//internal:jasmine_node_test.bzl", "jasmine_node_test")
 load("//internal:node_install.bzl", "node_repositories")
 load("//internal:npm_install.bzl", "npm_install")
```

Comment:
I'd like to drop `npm_install`. I don't think we need both, it will be more work to maintain both, and it's broken.

A polite way to roll that out would be to keep `npm_install` in this file but make it an alias of the new yarn impl

Or do we have to keep it for users who have a package-lock.json for their regular workflow and want Bazel to manage a copy of all the dependencies?

Reply:
Yeah. The rules are slightly different. yarn_install takes a yarn_lock attribute and npm_install takes a package_lock_json attribute. For users wanting to use an npm lock file they'll want to keep using npm_install.

Reply:
I think we should at least prevent anyone new from depending on `npm_install` if we're not going to make it deterministic yet. I don't want users to fall into an unhappy path by accident just because they started with a `package-lock.json` file